server/routes/user.js

```diff
@@ -28,4 +28,36 @@ router.post(
   }
 );
 
+router.put("/:id/experience", async (req, res) => {
+  const languages = { ...req.body };
+  const user = await User.findById(req.params.id);
+  const availableLanguages = [
```

Comment:
probably want this list to be as a constant somewhere else in case we need to reuse it